src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -106,6 +114,13 @@ A popular library based on observables and reactions.
 * [Getting started with MobX.dart][]
 * [Flutter: State Management with Mobx][], a video by Paul Halliday
 
+## Flutter Commands
+
+Reactive state management that uses the Command Pattern and is based on `ValueNotifiers`. Best in combination with [get_it](#getit), but can be used with `provider` or other locators too.
+
+* [flutter_command package][] 
+* [rx_command package][], `Stream` based implementation.
+
```

Comment:
Since you're saying "Best in combination" anyway, I suggest collapsing this into GetIt, as one of the `* ...` list items. This way, we keep this list nice and short, and it makes the GetIt option a little less sparse.

I know the `flutter_command` package is also useful without GetIt, but that doesn't mean it automatically gets its own section. If you feel strongly about this, let me know — I'm open to discussion.

Reply:
I think the Command pattern represents a very different approach than the others, therefore I think it's own section is justified.